test/integration/scripting_spec.js

```diff
@@ -280,4 +289,76 @@ describe("Interaction", () => {
       );
     });
   });
+
+  describe("in doc_actions.pdf for printing", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("doc_actions.pdf", "#\\34 7R");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must execute WillPrint and DidPrint actions", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          if (process.platform === "win32" && browserName === "firefox") {
+            // Doesn't work because of bug 1662471
+            return;
```

Comment:
Sorry, I just noticed this now and that's *really not* how you *temporarily* disable a test, since that pretty much guarantees that no one will ever fix it.
The correct way, since we're using Jasmine, is to add a `pending()` statement; i.e.
```javascript
pending("Disabled in Firefox on Windows, because of bug 1662471.");
```

Reply:
Good point, I also missed this one. @calixteman Let's address this one in a follow-up PR.